knowledge_graph/querying.py

```diff
@@ -18,6 +18,10 @@ class AttributeMissing(Exception):  # noqa
     pass
 
 
+class NodeTitleMissing(Exception):  # noqa
```

Comment:
private / expose it in the api __init__.py file

Reply:
nniu - the class is exposed because everything in querying.py is exposed - or am i missing something?